### PR TITLE
add RWTH results for Librispeech

### PR DESCRIPTION
**Pull request by christophmluscher**

### CODE REVIEW

**Comment by syhw** [comment]
Maybe expand slightly on the Notes / specificities of the system?

**Comment by buriy** [comment]
I also think they've made several interesting experiments:
+LSTM instead of Transformer rescoring, 
their attention+BPE+{LSTM or transformer} is also an interesting approach.
@syhw What do you think, is it a point to include multiple well-done experiments in such cases?
Librispeech is very specific, and its results aren't directly applicable to other more complicated domains, so just improving SOTA by less than 2% looks pretty random to me.
So I value the approaches more than the exact results.

**Comment by syhw** [comment]
It's OK to have a few lines of result for a given paper if they're interesting/important. (But please justify the specifics in the notes.)

**Comment by syhw** [comment]
It's OK to have a few lines of result for a given paper if they're interesting/important. (But please justify the specifics in the notes.)

**Comment by christophmluscher** [comment]
I added the mentioned model and expanded on the notes

File: README.md [hunk 1]
```diff
@@ -10,10 +10,12 @@ WER are we? An attempt at tracking states of the art(s) and recent results on sp
...
 | 2.5%  | 5.8%   | [SpecAugment: A Simple Data Augmentation Method for Automatic Speech Recognition](https://arxiv.org/abs/1904.08779) | April 2019 | Listen Attend Spell |
 | 5.83% | 12.69% | [Deep Speech 2: End-to-End Speech Recognition in English and Mandarin](http://arxiv.org/abs/1512.02595v1) | December 2015 | *Humans* |
 | 3.19% | 7.64% | [The CAPIO 2017 Conversational Speech Recognition System](https://arxiv.org/abs/1801.00059) | April 2018 | TDNN + TDNN-LSTM + CNN-bLSTM + Dense TDNN-LSTM across two kinds of trees
 | 3.80% | 8.76%  | [Semi-Orthogonal Low-Rank Matrix Factorization for Deep Neural Networks](http://www.danielpovey.com/files/2018_interspeech_tdnnf.pdf) | Interspeech, Sept 2018 |[Kaldi recipe](https://github.com/kaldi-asr/kaldi/blob/master/egs/librispeech/s5/local/chain/tuning/run_tdnn_1d.sh), 17-layer TDNN-F + iVectors|
+| .%  | .%   | [RWTH ASR Systems for LibriSpeech: Hybrid vs Attention](https://arxiv.org/abs/1905.03072) | May 2019 | encoder-attention-decoder + BPE + Transformer LM (no data augmentation) |
```
Comment (syhw) [inline README.md]: numbers are missing

Comment (christophmluscher) [reply README.md]: done. sry for the oversight

**Pull request merged**